.github/ISSUE_TEMPLATE/add_conference.yml

```diff
@@ -11,27 +11,39 @@ body:
       placeholder: <CONFERENCE NAME>
     validations:
       required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: Summary
+      description: Provide a summary (2-5 sentences) of the announcement information
+      placeholder: The <CONFERENCE> conference is happening in <LOCATION> on <DATE>. The CFP is open until <DATE>.
```

Comment:
All of the information in this placeholder is covered by dedicated form fields. If this is all we're asking, it seems like we should allow this field to be blank and default that kind of text instead of duplicating it.

Reply:
There was a part of me that was wanting that bottom information only so that it can be pushed into organizers content (if there is still hope that will happen one day). I would be okay with just removing the bottom information. @jonafato What are your thoughts on that?